packages/office-ui-fabric-react/src/components/Slider/Slider.base.tsx

```diff
@@ -109,7 +114,30 @@ export class SliderBase extends BaseComponent<ISliderProps, ISliderState> implem
               {originFromZero && (
                 <span className={css(classNames.zeroTick)} style={this._getStyleUsingOffsetPercent(vertical, zeroOffsetPercent)} />
               )}
-              <span ref={this._thumb} className={classNames.thumb} style={this._getStyleUsingOffsetPercent(vertical, thumbOffsetPercent)} />
+              {typeof marks === 'boolean' && marks && this._addTickmarks(css(classNames.regularTick))}
+              {thumblabel ? (
+                <TooltipHost
+                  content={'' + value}
+                  id={this._hostId}
+                  calloutProps={{ gapSpace: 5, beakWidth: 8, target: `#${this._buttonId}`, doNotLayer: true }}
```

Comment:
This values seem so random… do we have a design spec for this new feature? **cc:** @betrue-final-final

Reply:
oh quick question, which values are you referring to?

Reply:
5 and 8

Reply:
@micahgodbolt suggested these values because the tooltip looked weird with a space for just one character

Reply:
And again… I would love to see the the specs in a toolkit first. This is a new feature and we want to avoid having it the code and not in the toolkit and vice versa.

The fact that Micah suggested the numbers does not change a confirmation from a designer on the numbers in the form of an existing redline in any form. We can connect offline on that matter.

cc: @betrue-final-final